### PR TITLE
Update capacity and reductions to account for CMS now being an adjust…

### PR DESCRIPTION
**Pull request by caoimheb**
…ment

### CODE REVIEW

File: app/services/data/get-reductions.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ module.exports = function (workloadOwnerId) {
   if (workloadOwnerId !== undefined) {
     whereObject.workload_owner_id = workloadOwnerId
   }
-  whereObject.contact_id = null
```
Comment (tswann) [inline app/services/data/get-reductions.js]: :+1:

File: app/services/get-capacity-view.js [hunk 2]
```diff
@@ -74,12 +74,18 @@ var parseCapacityBreakdown = function (workloadReports, organisationLevel) {
 }
 
 var buildCapacityBreakdownEntry = function (workloadReport) {
+  var cmsPercentageValue = null
+
+  if (workloadReport.cmsAdjustmentPoints > 0) {
```
Comment (tswann) [inline app/services/get-capacity-view.js]: 🤔  What if we have a net negative adjustments total? Could happen if someone is the recipient of any `-points` values.

Comment (caoimheb) [reply app/services/get-capacity-view.js]: Myself, Terence and Andy had a chat about this. The decision was that if an OM has had points removed via CMS, then none of their remaining total_points are due to CMS work. So the percentage of their capacity which is being taken up by CMS is 0%. Thoughts?

Comment (tswann) [reply app/services/get-capacity-view.js]: Makes sense 👍

**Pull request merged**